airflow/models/dataset.py

```diff
@@ -0,0 +1,53 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+from typing import Dict, Optional
+from urllib.parse import urlparse
+
+from sqlalchemy import Column, Index, Integer
+
+from airflow.models.base import Base, StringID
+from airflow.utils import timezone
+from airflow.utils.sqlalchemy import ExtendedJSON, UtcDateTime
+
+
+class Dataset(Base):
+    """A table to store datasets."""
+
+    id = Column(Integer, primary_key=True, autoincrement=True)
+    uri = Column(StringID(length=500), nullable=False)
+    extra = Column(ExtendedJSON, nullable=True)
+    created_at = Column(UtcDateTime, default=timezone.utcnow(), nullable=False)
+    updated_at = Column(UtcDateTime, default=timezone.utcnow(), onupdate=timezone.utcnow(), nullable=False)
```

Comment:
I didn't know about `onupdate` in sqlalchemy

Reply:
til :)

Reply:
```suggestion
    updated_at = Column(UtcDateTime, default=timezone.utcnow,, onupdate=timezone.utcnow, nullable=False)
```

I think? Otherwise you've set the default to one specific date time

Reply:
whoa :) 

this was a copy paste from https://github.com/apache/airflow/blob/2258e13cc78faec80054c223eca9378fd33e18fd/airflow/models/sensorinstance.py#L62-L63